webgpu.h

```diff
@@ -1539,6 +1565,12 @@ typedef struct WGPUSurfaceConfiguration {
      * When and in which order the surface's frames will be shown on the screen. Defaults to @ref WGPUPresentMode_Fifo.
      */
     WGPUPresentMode presentMode;
+    /**
+     * Specifies how color values are displayed to the screen.
+     *
+     * Implementation support is not required for @ref WGPUSurfaceToneMappping_Extended.
+     */
+    WGPUSurfaceToneMappingMode toneMappingMode;
```

Comment:
ditto, should we expose this in surface capabilities?

Reply:
I think this has the same issue as #362 though the interaction is a bit less complex, it's pretty likely that, if the system can do HDR for any format, then all formats that support values>1 (so, float formats, and the hypothetical [XR formats](https://github.com/gpuweb/gpuweb/issues/4108)) would support it.

I'll open a separate issue to track it, and we can start with this.

Reply:
#366